tests/cxx/template_default_args_roundtrip.cc

```diff
@@ -0,0 +1,32 @@
+//===--- template_default_args_roundtrip.cc - test input file for iwyu ----===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+
```

Comment:
It would be nice with a prose comment here mentioning the example with std::hash<T> as default argument, it's a good example to frame the problem.